singleuser/user-fixes.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+custom_path = os.path.expanduser('~/user-fixes.py')
+if os.path.exists(custom_path):
+    with open(custom_path, 'r') as f:
```

Comment:
Use `rb` and `compile` will understand any encoding the file uses (if it's complying with the Python standard).